docs/Documentation/UserGuide/3-Server/1-Download.md

```diff
@@ -34,7 +34,7 @@ IoTDB provides you three installation methods, you can refer to the following su
 To use IoTDB, you need to have:
 
 1. Java >= 1.8 (Please make sure the environment path has been set)
-2. Maven >= 3.1 (If you want to compile and install IoTDB from source code)
+2. Maven >= 3.1 (If you want to compile and install IoTDB from source code. You don't need this if you install from binary files)
```

Comment:
This is not what I mean. We have a script that can replace maven.
![image](https://user-images.githubusercontent.com/23610645/67997614-cbba0700-fc8f-11e9-8b31-88113eb35ee9.png)
And notify the users: "If you don't have maven installed, you should replace 'mvn' in the following commands with 'mvnw.sh' or 'mvnw.cmd'".

Reply:
> This is not what I mean. We have a script that can replace maven.
> ![image](https://user-images.githubusercontent.com/23610645/67997614-cbba0700-fc8f-11e9-8b31-88113eb35ee9.png)
> And notify the users: "If you don't have maven installed, you should replace 'mvn' in the following commands with 'mvnw.sh' or 'mvnw.cmd'".

Thank you, it makes sense.